signup.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="signup.css">
+    <title>login</title>
+</head>
+<body>
+    <div class="logo_box">
+        <img class="logo" src="images/login/logo.png">
+        <div class="login_box">
+            <div id="email">
+                <span class="email">이메일</span>
+                <h2 class="email_input"></h2>
+            </div>
+            <div id="nickname">
+                <span class="nickname">닉네임</span>
+                <h2 class="nickname_input"></h2>
+            </div>
+            <div id="password">
+                <span class="password">비밀번호</span>
+                <h2 class="password_input"></h2>
+            </div>
+            <div id="password_check">
+                <span class="password_check">비밀번호 확인</span>
+                <h2 class="password_check_input"></h2>
+            </div>
+            <h3 class="signup_button">
+                회원가입
+            </h3>
```

Comment:
❗️ 수정요청
해당 요소는 추후 input의 값들을 제출하는 역할을 하므로 button 태그로 작성해주세요~

```suggestion
            <button type="submit" class="signup_button">
                회원가입
            </button>
```